### PR TITLE
get data from meta leader

### PR DESCRIPTION
**Pull request by panda-sheep**
For some reasons, the data read from different copies of meta data may be inconsistent.
Therefore, we uniformly obtain data from the leader.

### CODE REVIEW

File: src/common/clients/meta/MetaClient.cpp [hunk 1]
```diff
@@ -814,11 +814,14 @@ MetaClient::submitJob(cpp2::AdminJobOp op, cpp2::AdminCmd cmd, std::vector<std::
     req.set_paras(std::move(paras));
     folly::Promise<StatusOr<cpp2::AdminJobResult>> promise;
     auto future = promise.getFuture();
-    getResponse(std::move(req), [] (auto client, auto request) {
+    getResponse(std::move(req),
```
Comment (Shylock-Hg) [inline src/common/clients/meta/MetaClient.cpp:817]: I think you could change the default argument value to `true` directly.

Comment (panda-sheep) [reply src/common/clients/meta/MetaClient.cpp:817]: make sense

**Review by liuyu85cn — APPROVED** [review]
LGTM

**Comment by darionyaphet** [comment]
Raft support strong consistent to make all peer hold the same data why read different peer could found different value ？

**Comment by Shylock-Hg** [comment]
> Raft support strong consistent to make all peer hold the same data why read different peer could found different value ？

Raft only make major consistent, I think so.

**Comment by critical27** [comment]
Good job. Since there are more work than this, for example, we don't need sync anymore, ActiveHostMan and LastUpdateTimeMan perhaps need to change as well. Would you move this PR to 1.0, let me handle 2.0.

**Comment by darionyaphet** [comment]
> > Raft support strong consistent to make all peer hold the same data why read different peer could found different value ？
> 
> Raft only make major consistent, I think so.

yep vote should more than half

**Review by critical27 — APPROVED** [review]

**Review by Shylock-Hg — APPROVED** [review]
Well done! 
Maybe we could remove the `toLeader` option and I don't think we need the inconsistent meta data that different from data itself.

**Pull request merged**